src/Components/Common/Sidebar/SidebarUserCard.tsx

```diff
@@ -15,8 +15,15 @@ const SidebarUserCard = ({ shrinked }: { shrinked: boolean }) => {
         shrinked ? "mx-auto flex-col" : "mx-5"
       } transition-all duration-200 ease-in-out`}
     >
-      <Link href="/user/profile" className="flex-none py-3">
-        <CareIcon icon="l-user-circle" className="text-3xl text-white" />
+      <Link href="/user/profile">
+        <img
+          src={user.read_profile_picture_url || "/images/empty_avatar.jpg"}
```

Comment:
Can't we add support for inital avatars as default? 

Example: https://github.com/pupilfirst/pupilfirst/blob/master/app/frontend/shared/Avatar.res

Reply:
Implemented in #8680